src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$main-f-color: #060b35;
+$secondary-f-color: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  color: $main-f-color;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)
when all elements have the same `box-sizing` value, their behavior becomes more predictable